docs/site/development/cldr-big-red-switch/commit-checker.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Commit Checker
+---
+
+# Commit Checker
+
+Commit checker is a tool to ensure that Jira tickets for the release are in sync with commits in Github.
+It can validate any tickets which have commits in GitHub,
+while tickets which are fixed elsewhere such as in the Survey Tool still must be validated manually.
+
+## How to run the Commit Checker
+
+1. Go to Actions · unicode-org/cldr (github.com)
+2. Select Workflow: Commit-Checker
+
+![Commit Checker Workflow](/images/development/commit-checker-workflow-fields.png)
+
+3. Fill out the fields:
+   - Branch: main
+     - Always use main for branch 
+   - CLDR Commit 
+     - Run against main 
+     - Use maint/maint-xx when the maint branch is created
+   - Fix version
+     - The current release version in development
+   - Last version
+     - Use the Dot release if the last released version of a Dot release. Use the Hyphen (e.g. 38-1)
+    Commit Checker ICU Repo/ ICU branch/tag
+4. Once the workflow completes, expand Run Check and Publish and find the report URL (e.g. https://cldr-smoke.unicode.org/***/CLDR-Report-YYYY-MM-DD-vXX-HASH.html)
```

Comment:
we could actually easily add a link in the workflow summary now.. file a ticket on it? 1-liner in the workflow.

Reply:
Sure, filed: https://unicode-org.atlassian.net/browse/CLDR-18684